config/src/main/java/com/typesafe/config/impl/SerializedConfigValue.java

```diff
@@ -355,7 +382,15 @@ private static AbstractConfigValue readValueData(DataInput in, SimpleConfigOrigi
             String sd = in.readUTF();
             return new ConfigDouble(origin, vd, sd);
         case STRING:
-            return new ConfigString.Quoted(origin, in.readUTF());
+            int read_times = 1;
+            if (longStr)
```

Comment:
How does longStr get set to true when reading?

Reply:
I set it to true when writing a long string. It applies to most write-first-read-next situations. Maybe there is better solutions.

Reply:
Also, we would appreciate it a lot if you could give us some feedback. For example, to what extent does the recommended fixation help? Do you like/need a fixation recommendation system? Your opinion means a lot to us. Thanks!